### ISSUE TITLE
[Question] Is there a reason UBC sedimentary stone was left out?

### ISSUE THREAD

**Issue by scribbler957** [asiekierka/UnlimitedChiselWorks#36]
I'm just wondering, I've got all this cool dolomite and stuff that doesn't appear to chisel.

**Comment by NickelHub** [comment]
Stone cannot be chiseled. UBC's sedimentary doesn't have any brick.

**Comment by asiekierka** [comment]
Yes. It doesn't have a stone brick or cobblestone variant, and Chisel doesn't support regular stone by itself.
